internal/common/foraging/deerhunt_test.go

```diff
@@ -0,0 +1,61 @@
+package foraging
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+var huntParticipants = map[shared.ClientID]float64{shared.Team1: 1.0, shared.Team2: 0.9} // arbitrarily chosen for test
+var params = DeerHuntParams{P: 0.95, Lam: 1.0}
+var hunt = DeerHunt{Participants: huntParticipants, Params: params}
```

Comment:
I tend to not like doing this even though it's convenient. It stops your tests from being able to be run in parallel. Test dependencies should be defined within the Test function and self-contained unless you are sure these are not changed at all.

Reply:
It also prevents the ability for `TestTotalInput` to benefit from a wider range of test inputs.

Reply:
Yeah tbh I actually meant to change this but just forgot - thanks!